charts/scalar-manager/Chart.yaml

```diff
@@ -5,13 +5,17 @@ type: application
 version: 2.0.0-SNAPSHOT
 appVersion: 2.0.0-SNAPSHOT
```

Comment:
Now we only have 1.0.0-SNAPSHOT images.
I use 2.0.0-SNAPSHOT here because we will bump the SNAPSHOT images version to 2.0.0 after officially release Scalar Manager.